WordPress/src/main/res/values/strings.xml

```diff
@@ -3353,7 +3353,7 @@ translators: sample content for "Services" page template -->
     <string name="backup_download">Backup Download</string>
     <string name="backup_download_details_page_title">Download Backup</string>
     <string name="backup_download_details_header">Create downloadable backup</string>
-    <string name="backup_download_details_description">%1$s %2$s is the selected point to create a downloadable backup.</string>
+    <string name="backup_download_details_description_with_two_parameters">%1$s %2$s is the selected point to create a downloadable backup.</string>
```

Comment:
This is not the best name, but I couldn't think of anything meaningful. If you have any ideas, I am happy to make the change.